drivers/gpu/drm/rp1/rp1-dsi/rp1_dsi_dsi.c

```diff
@@ -145,11 +145,18 @@
 #define DSI_PHYRSTZ_FORCEPLL_LSB 3
 #define DSI_PHYRSTZ_FORCEPLL_BITS  BIT(DSI_PHYRSTZ_FORCEPLL_LSB)
 
+#define DPHY_CLK_PN_SWAP		0x35
```

Comment:
I could understand putting this register out in front if the other registers were in numerical order, but they aren't.

Reply:
Registers now in order, and lots of whitespace issues and other cruft sorted.